apitest/testNgXmlFiles/residentApi.xml

```diff
@@ -340,29 +340,38 @@
 			<class name="io.mosip.testrig.apirig.testscripts.SimplePost" />
 		</classes>
 	</test>
-	<test name="GetIdentityMapping">
+	<test name="GetPartnersByPartnerType">
```

Comment:
why below 9 lines are added newly? Can you replace it with upstream revision if it is not intended

Reply:
Replaced with upstream revision